### PR TITLE
Feature/migrations update

### PR DESCRIPTION
**Pull request by jaalazard**
## Description

Please include a summary of the change or which issue is fixed.

## Type of change

- [x] New feature (non-breaking change which adds functionality)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Area of change

- [ ] Frontend
- [x] Backend
- [ ] Other: .....

## General checklist:

- [x] My code follows the style guidelines of this project
- [x] I ran `bun run format`/`bun run lint` before commit
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation where needed
- [x] My changes generate no new warnings
- [x] I have checked my code and corrected any misspellings
- [x] I have updated `develop` and merged to my branch before submitting pull request
- [x] My pull request generate no conflicts with `develop` branch
- [x] I requested code review from other team members

## Frontend checklist:

- [ ] I followed guidelines for `HTML`/`JSX`/`Typescript`
- [ ] My `Javascript` generate no new console errors
- [ ] I tested my code cross browsers
- [ ] My slice is pixel perfect for both desktop and mobile according to design
- [ ] I conducted basic QA to assure all features are working
- [ ] I tested responsive for mobile and tablet resolutions

## Backend checklist:

- [x] I tested admin by manually adding content from zero
- [x] I created easy to use admin experience which is self-explanatory
- [x] I added description to admin fields in hard-to-understand areas
- [x] I followed guidelines for naming `Typescript` variables
- [x] I conducted basic QA to assure all features are working

### CODE REVIEW

**Review by No3ml3 — DISMISSED** [review]

**Review by Cardamced — DISMISSED** [review]

File: packages/backend/migrations/src/migrations/20240206085522-topping-flavour.ts [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+  await db.transaction().execute(async (trx) => {
+    await sql`
+      ALTER TABLE topping
+      DROP COLUMN flavour;
+    `.execute(trx);
+
+    await sql`
+      ALTER TABLE topping
+      ADD COLUMN flavour ENUM('fruity', 'spicy', 'earthy', 'herbaceous', 'floral', 'woody', 'bitter', 'sweet', 'salty', 'peppery', 'mixed', 'fresh', 'acidulous', 'sour', 'neutral', 'umami', 'astringent', 'other') NOT NULL
+    `.execute(trx);
```
Comment (YDrogen) [inline packages/backend/migrations/src/migrations/20240206085522-topping-flavour.ts]: Fact : Que tu `up` ou `down`, au final on fait la même chose
Question : Quel est l'intérêt de la migration ?

Comment (jaalazard) [reply packages/backend/migrations/src/migrations/20240206085522-topping-flavour.ts]: Le but était de changer l'enum

Comment (YDrogen) [reply packages/backend/migrations/src/migrations/20240206085522-topping-flavour.ts]: Mais là, y'a rien qui change entre le up et le down, du coup la migration sert à rien ?

Comment (jaalazard) [reply packages/backend/migrations/src/migrations/20240206085522-topping-flavour.ts]: Ah oui, au temps pour moi, c'est un peu contre-intuitif de remettre la version pétée, mais c'est logique

**Review by Cardamced — APPROVED** [review]

**Review by Justicesvr — APPROVED** [review]

**Review by YDrogen — APPROVED** [review]

**Pull request merged**